src/controller/model/device.ts

```diff
@@ -137,6 +137,11 @@ class Device extends Entity {
         return this.endpoints.find((e): boolean => e.ID === ID);
     }
 
+    // There might be multiple endpoints with same DeviceId but it is not supported and first endpoint is returned
+    public getEndpointByDeviceId(devId: number): Endpoint {
```

Comment:
Instead of a number, would it be nicer to pass a string down here? e.g. `onOffLight` instead of `0x0000`. The lookup can be added here: https://github.com/Koenkk/zigbee-herdsman/tree/master/src/zcl/definition

Reply:
Yes, string lookup would be more consistent with the rest of the code